CommonMC/fcl/prolog.fcl

```diff
@@ -5,6 +5,8 @@
 
 BEGIN_PROLOG
 
+TIME_MAPS : [ protonTimeMap, muonTimeMap, cosmicTimeMap ]
```

Comment:
Why move this outside CommonMC namespace?  The point of namespaces is to provide provenance, moving the content outside the namespace defeats that.

Reply:
This definition has been moved back into the CommonMC namespace.